### DISCUSSION TITLE
A lightweight version without electron

### DISCUSSION THREAD

**Discussion by zhulik** [timmo001/system-bridge#393] [Ideas] [answered]
# Description

Thanks for a great project.
The other day I was working on a PKGBUILD(arch linux build system) for this app. And I thought that it would be amazing to have a lightweight version of system-bridge that doesn't have electron's ~500Mb overhead and that can be run as a daemon or service. Even if it would miss some of the electron app's functionality.

WDYT?

**Comment by timmo001** [comment]
I like the idea, it would require a refactor of the application, so it can be run in this way, removing the electron dependency etc. before adding too many settings to the app, to not introduce a lot of breaking changes.

Also discussed in #360

**Comment by timmo001** [comment, marked as answer]
Recently rewritten the application to not use electron. TBR soon™
